runner/__init__.py

```diff
@@ -216,8 +229,13 @@ def main():
     parser = make_argument_parser()
     args = parser.parse_args()
     logging.basicConfig(format="%(asctime)s - %(levelname)s - %(message)s", level=args.loglevel)
-
     config = Config()
+
+    if args.syslog:
```

Comment:
This will fail on Windows because parser.add_argument("--syslog", ...) is for linuix/mac only